beacon-chain/sync/service.go

```diff
@@ -196,12 +196,6 @@ func (s *Service) Stop() error {
 
 // Status of the currently running regular sync service.
 func (s *Service) Status() error {
-	if s.chainStarted.IsNotSet() {
-		return errors.New("chain not yet started")
-	}
-	if s.initialSync.Syncing() {
```

Comment:
This was duplicated and confusing. Initial sync already projected error here: 
https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/sync/initial-sync/service.go#L131